DocumentService/Repositories/IDocumentRepository.cs

```diff
@@ -42,13 +43,13 @@ public interface IDocumentRepository
         /// </summary>
         /// <param name="documentInfo">Document object with updated data</param>
         /// <returns>True if successful</returns>
-        Task<bool> Update(DocumentInfo documentInfo);
+        Task<bool> Update(DocumentDTO documentDTO, Guid id, HttpContext httpContext);
```

Comment:
We should not be using HttpContext in the DocumentRepository. The Repository should not have to have knowledge of the HttpContext.
```suggestion
        Task<bool> Update(DocumentDTO documentDTO, Guid id);
```